presto-server-rpm/src/main/rpm/postremove

```diff
@@ -1,9 +1,13 @@
 # Post erase script
 
-# Delete the conf directory manually during uninstall.
-# rpm -e wont remove it, because this directory was manually updated in postinstall
-rm -rf /etc/presto
-# Delete the data directory manually during uninstall.
-# rpm -e wont remove it, because this directory may later contain files not
-# deployed by the rpm
-rm -rf /var/lib/presto
+# if this is the last version of presto-server-rpm being removed (i.e. not on upgrade)
+if [ "$1" -eq 0 ]
```

Comment:
$1 in the previous example, which holds a count of the number of versions of the package that are installed